src/metaxy/models/feature_spec.py

```diff
@@ -186,6 +209,7 @@ def __init__(
         deps: list[FeatureDep] | None = None,
         fields: list[FieldSpec] | None = None,
         id_columns: list[str] | None = None,
+        metadata: dict[str, Any] | None = None,
```

Comment:
Should use  https://docs.pydantic.dev/2.0/usage/types/json/ (`JSON[dict[str, Any]]` ?)

Reply:
fixed now